platform/src/main/java/org/stellar/anchor/platform/payment/observer/stellar/StellarPaymentObserver.java

```diff
@@ -25,12 +26,18 @@
 import org.stellar.sdk.requests.PaymentsRequestBuilder;
 import org.stellar.sdk.requests.RequestBuilder;
 import org.stellar.sdk.requests.SSEStream;
+import org.stellar.sdk.responses.Page;
 import org.stellar.sdk.responses.operations.OperationResponse;
 import org.stellar.sdk.responses.operations.PathPaymentBaseOperationResponse;
 import org.stellar.sdk.responses.operations.PaymentOperationResponse;
 import shadow.com.google.common.base.Optional;
 
 public class StellarPaymentObserver implements HealthCheckable {
+  /** The maximum number of results the Stellar Blockchain can return. */
+  private static final int MAX_RESULTS = 200;
```

Comment:
Can we create a ticket to make this a config value?

Reply:
I can, but why would we want to make this configurable? The most efficient thing would be to perform the least amount of requests to the blockchain, which is achievable with `?limit=200`.